src/logic/basic.lean

```diff
@@ -946,6 +946,9 @@ theorem exists_eq {a' : α} : ∃ a, a = a' := ⟨_, rfl⟩
 
 @[simp] theorem exists_eq' {a' : α} : ∃ a, a' = a := ⟨_, rfl⟩
 
+@[simp] theorem exists_unique_eq {a' : α} : ∃! a, a' = a :=
+by simp only [exists_unique, and_self, forall_eq', exists_eq']
```

Comment:
I think this should be
```suggestion
@[simp] theorem exists_unique_eq' {a' : α} : ∃! a, a' = a :=
by simp only [exists_unique, and_self, forall_eq', exists_eq']
```
to match the order of `exists_eq'` above, which suggests we also should have
```lean
@[simp] theorem exists_unique_eq {a' : α} : ∃! a, a = a' := sorry
```

Reply:
Thanks Eric. Done I believe -- exists_eq only has the quote one as a simp lemma, hopefully that was correct here too (only added `exists_unique_eq'` as simp). Otherwise let me know. Appreciated!